### DISCUSSION TITLE
What kind of kafka span is endpoint?

### DISCUSSION THREAD

**Discussion by 807801002** [apache/skywalking#7990] [Q&A]
I have lots of kafka type span,but just a fraction of kafka span shown as endpoint, What kind of kafka span is marked as endpoint?

**Comment by wu-sheng** [comment]
I think all of them.

**Reply by 807801002** [reply]
I have a trace named Kafka/shopstock-bizrecord-hbase/Consumer/group_shopstock-bizrecord-hbase",but i cannot find "Kafka/shopstock-bizrecord-hbase/Consumer/group_shopstock-bizrecord-hbase" at the endpoint select list when I selected the "shopstock-service" service
<img width="669" alt="Snipaste_2021-10-22_15-56-59" src="https://user-images.githubusercontent.com/8719537/138416655-38b88a87-4173-476b-9ef6-fda4055e15b8.png">
<img width="957" alt="Snipaste_2021-10-22_16-10-53" src="https://user-images.githubusercontent.com/8719537/138418563-0dcc09e9-af9a-41b1-bfbc-f967d57a60e2.png">

There is only 4 kafka type endpoints in "shopstock-service" service

**Reply by wu-sheng** [reply]
Could you check whether your endpoint search working with backend? In some old release, there was a bug. 
If network request(GraphQL) calls OAP, check `endpoint_traffic` in your chosen storage.
